content/en/ebpf.md

```diff
@@ -0,0 +1,27 @@
+---
+title: eBPF
+status: Completed
+category: architecture
+---
+
+## What it is
+
+Linux system has two spaces: kernel and user space. Kernel space is the core of the operating system and it is the only part that has unlimited access to the hardware. 
+Usually, applications reside in the user space and if they need some higher permissions, they send a request to the kernel space for that.
+For applications that require more flexibility (exp. to access hardware), there is an approach called “Linux kernel modules”. It allows extending the kernel base without adding directly to the kernel source code. 
+However, it also introduces some security risks.
```

Comment:
```suggestion
However, such an approach also introduces some security risks.
```

Reply:
Risk applies to other approaches (non-eBFP) or all these approaches?

Reply:
I added some edit suggestions about this.